Apps/Src/Temperature.c

```diff
@@ -225,12 +240,26 @@ ErrorStatus Temperature_UpdateAllMeasurements(){
 SafetyStatus Temperature_CheckStatus(uint8_t isCharging){
 	int32_t temperatureLimit = isCharging == 1 ? MAX_CHARGE_TEMPERATURE_LIMIT : MAX_DISCHARGE_TEMPERATURE_LIMIT;
 
+	// const int32_t CHANNELS_IN_LAST_MINION = MAX_VOLT_SENSORS_PER_MINION_BOARD - (NUM_MINIONS * MAX_VOLT_SENSORS_PER_MINION_BOARD - NUM_BATTERY_MODULES);
+	// const int32_t MAX_TEMP_CHANNELS = MAX_VOLT_SENSORS_PER_MINION_BOARD;
+
+	// volatile SafetyStatus retVal = SAFE;
+
 	for (int i = 0; i < NUM_MINIONS; i++) {
+		// if there are less than 16 temperature sensors, plug them in in order of which channel they are on
+		// for example, if there are 14 sensors, use sensor numbers (1-indexed) 1-7 and 9-15
+		// int numChannels = (i + 1 < NUM_MINIONS) ? MAX_TEMP_CHANNELS : CHANNELS_IN_LAST_MINION;
+		// for (int j = 0; j < numChannels; j++) {
 		for (int j = 0; j < MAX_TEMP_SENSORS_PER_MINION_BOARD; j++) {
 			if (i * MAX_TEMP_SENSORS_PER_MINION_BOARD + j >= NUM_TEMPERATURE_SENSORS) break;
-			if (ModuleTemperatures[i][j] > temperatureLimit) {
+			if ((temperatures[i][j] > temperatureLimit) || (temperatures[i][j] == TEMP_ERR_OUT_BOUNDS)) {
 				return DANGER;
 			}
+			/*
```

Comment:
Same question as before - since all of this is commented, are we keeping this around just in case we need it in the future?

Reply:
I'll take this out. We aren't using it anymore